content/blog/2020-01-29-winter-school-recap.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Report from the URSSI Winter School pilot"
+date: 2020-01-29
+author: "Karthik Ram, Kyle Niemeyer, and Jeffrey Carver"
+---
+
+![Winter School](https://imgur.com/njacNOL.png)
+
+
+<br><br>
+
+In late December 2019, we ran our first ever [URSSI school on research software engineering](https://github.com/si2-urssi/winterschool#tentative-schedule). We began accepting applications in July and received an overwhelming response to our call for applications. For the 30 participant slots available, we received 169 applications, meaning we had a challenging time selecting the participants and had to turn away a large number of interested researchers. Our students came from 22 different universities/research institutes, mostly graduate students (19 students, 8 postdocs, and 1 professional researcher), and from fields such as astrophysics, bioinformatics, chemical engineering and neuroscience. While the selection committee used multiple criteria to evaluate and select participants, successful applicants already had some experience with Python programming, Git, and Unix skills, which was necessary to benefit from the workshop.
+
+Our goal wasn't to repeat the same material covered in bootcamps and Software Carpentry style workshops, but to focus on the research software engineering skills that are not formally taught in any setting. These skills include best practices for converting a loose collection of scripts as installable software, testing, collaborative software development, code review, and related topics such as licensing and archiving.  Two months before the workshop, we asked each student to select a software project related to their research to bring to the workshop. Our intent was to intermingle instructions with work time for students to apply these skills to their own projects. All of our materials are available on [URSSI's GitHub](https://github.com/si2-urssi/winterschool#tentative-schedule).
```

Comment:
for what it's worth, some of these topics are taught in other places, such as the HEP training offered by the HEP software foundation, MolSSI, and Katy previously developed a testing module for software carpentry, though it's not hosted by them.  Maybe change "not formally taught in any setting" to "generally not formally taught in most settings"